src/logging.cc

```diff
@@ -779,7 +779,8 @@ static void WriteToStderr(const char* message, size_t len) {
 }
 
 inline void LogDestination::MaybeLogToStderr(LogSeverity severity,
-					     const char* message, size_t message_len, size_t /*prefix_len*/) {
+					     const char* message, size_t message_len, size_t prefix_len) {
```

Comment:
The parameter name is uncommented but still unused. What's the purpose of this change then?

Reply:
For Android. https://github.com/google/glog/blob/9733727be4aed835e21a8f73c403f546b71663fe/src/logging.cc#L798

Reply:
I see.  Could you please then move the `void` expression to the `#if` block that does not use the parameter?